ledger/db/src/lib.rs

```diff
@@ -659,13 +650,16 @@ impl LedgerDB {
             }
         }
 
-        // A block must have outputs.
-        if block_contents.outputs.is_empty() {
+        // A block must have outputs, unless it has minting-related transactions.
+        let has_minting_txs =
```

Comment:
Note from live review - would we want to worry about the requirement that a MintTx does need to include outputs?

Reply:
Seems like a good improvement, I will be adding that.